06-building-a-program-start-to-finish/writing-tests/output/spec.js

```diff
@@ -1,4 +1,4 @@
-/* My Test Setups */
```

Comment:
Rename this file. Please see your peers for the preferred naming standards for test files.